### PR TITLE
chore: lodash tree shaking imports

### PR DESCRIPTION
**Pull request by rtpa25**
- closes https://github.com/hyperdxio/hyperdx/issues/36

### CODE REVIEW

**Comment by wrn14897** [comment]
We are going to add this to the guide soon for the better readability (ideally enforce by linting). Can you apply the following pattern on modules import ? Basically imports are separated into 3 chunks by a new line:
1. built-in
2. 3rd-party
3. internal

And each chunk of imports are sorted alphabetically.

File: packages/api/src/clickhouse/index.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import * as fns from 'date-fns';
 import SqlString from 'sqlstring';
-import _ from 'lodash';
+import map from 'lodash/map';
```
Comment (wrn14897) [inline packages/api/src/clickhouse/index.ts:3]: I think we don't need to do the tree shaking for files in `api/`.

File: packages/api/src/utils/logger.ts [hunk 2]
```diff
@@ -1,18 +1,14 @@
...
 
 import {
   APP_TYPE,
   HYPERDX_API_KEY,
   INGESTOR_API_URL,
-  IS_DEV,
   IS_PROD,
 } from '../config';
 
-import type { IUser } from '../models/user';
```
Comment (wrn14897) [inline packages/api/src/utils/logger.ts:14]: 👍

File: packages/api/src/clickhouse/index.ts (hunk 1 above)
Comment (rtpa25) [reply packages/api/src/clickhouse/index.ts:3]: I will assume, bundle size is not a concern for server side, as you are not deploying onto a serverless environment. Please correct me if am wrong

**Comment by rtpa25** [comment]
Taken care of the import order issues

File: packages/api/src/clickhouse/index.ts (hunk 1 above)
Comment (wrn14897) [reply packages/api/src/clickhouse/index.ts:3]: Correct, its not for the serverless environment

**Comment by wrn14897** [comment]
Things look good. Just need to resolve the conflicts and rollback changes in `/api` :)

**Review by wrn14897 — APPROVED** [review]
LGTM. Thanks for cleaning things up!

**Pull request merged**